inffast.c

```diff
@@ -12,6 +12,14 @@
 #include "functable.h"
 
 
+/* Behave like chunkcopy, but avoid writing beyond of legal output. */
+static inline uint8_t* chunkcopy_safe(uint8_t *out, uint8_t const *from, unsigned len, uint8_t *safe) {
+    unsigned safelen = (unsigned)((safe - out) + 1);
+    len = MIN(len, safelen);
+    memcpy(out, from, len);
+    return out + len;
+}
+
```

Comment:
Duplicated code, should be moved to a header.

Reply:
I had thought of that but wasn't sure how I felt about adding yet another header for one function.  Is there an existing header where it belongs better?

Reply:
@KungFuJesus `inflate_p.h` would be maybe best existing one...

Reply:
Will do.  Now I just have to track down the failing pigz tests...I don't get segfaults just bad crc32s.